MAINTAINERS.md

```diff
@@ -14,12 +14,12 @@ Maintainers are assigned the following scopes in this repository:
 
 <!-- Please keep this sorted alphabetically by github -->
 
-| Name |   GitHub ID   | Scope | LFID |    Discord ID    | Email | Company Affiliation |
-|------|---------------|-------|------|------------------|-------|---------------------|
-|      | rbarker-dev   |       |      | djbbot           |       | Hashgraph           |
-|      | andrewb1269hg |       |      | andrewb1269      |       | Hashgraph           |
-|      | nathanklick   |       |      | nathan.hashgraph |       | Hashgraph           |
-|      | jjohannes     |       |      |                  |       |                     |
+|       Name       |   GitHub ID   | Scope | LFID |    Discord ID    | Email | Company Affiliation |
+|------------------|---------------|-------|------|------------------|-------|---------------------|
+| Roger Barker     | rbarker-dev   |       |      | djbbot           |       | Hashgraph           |
+| Andrew Brandt    | andrewb1269hg |       |      | andrewb1269      |       | Hashgraph           |
+| Nathan Klick     | nathanklick   |       |      | nathan.hashgraph |       | Hashgraph           |
+| Jendrik Johannes | jjohannes     |       |      |                  |       | Gradle              |
```

Comment:
```suggestion
| Jendrik Johannes | jjohannes     |       |      |                  |       |                     |
```

I am not affiliated with the company Gradle. ATM, I do not work for any company directly (sole proprietorship / freelancer). Can we just leave this empty?

Reply:
For sure.. let me do that.